integrated_channels/canvas/docs/decisions/0001-integration-classic-vs-lti.rst

```diff
@@ -0,0 +1,37 @@
+Canvas Integration via Classic Integration model vs LTI
+=======================================================
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+We would like to capture points about the decision to choose the Class integration model
+over LTI model, in order to make edX courses available in Canvas LMS for partner institutions.
+
+Decision
+--------
+
+For Canvas integration, these following use cases are prioritized:
+
+* Students can easily discover courses within their institution’s online learning experience
+* Faculty can easily view student performance data within their institution’s online learning experience
+
+There is another use case that is deemed not applicable w.r.t Canvas integration:
```

Comment:
Could maybe reclassify this as "Out of scope for Canvas integration", took me a few read throughs to get your intent here but could also be fine and I just haven't had enough coffee this morning